src/extractors/extractors/src/extractors/sketch.ts

```diff
@@ -251,7 +251,7 @@ class SketchExtractor implements Extractor {
     await createFolders(assetsDirectory, [AssetFolder.Slice]);
     reporters.progress(`Running sketchtool export commands for ${assetName}`);
     const [rawDump] = await Promise.all([
-      execAsync(`${parserCliPath} dump ${source}`, {maxBuffer: 48 * (1 << 20)}),
+      execAsync(`${parserCliPath} dump ${escapeShell(source)}`, {maxBuffer: Infinity}),
```

Comment:
This is the command we run to get a dump from the Sketch file, this dump is a big JSON file. @zackbrown regarding `Infinity`: alternatively we could use an arbitrarily high number, but seems safer do this to cover all possible file sizes. What do you think?

Reply:
`Infinity` seems fine to me 👍